sbi/neural_nets/density_estimators/base.py

```diff
@@ -48,11 +72,66 @@ def loss(self, input: Tensor, condition: Tensor, **kwargs) -> Tensor:
     def sample(self, sample_shape: torch.Size, condition: Tensor, **kwargs) -> Tensor:
         r"""Return samples from the density estimator.
 
+        Note:
```

Comment:
Arg description for `sample_shape` and `condition` is missing.

Reply:
Should be there by I will move the Notes down